book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] `rental` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.